doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,241 +63,400 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
-| city | String | Yes |  |
-| resource_pool_name | String | Yes |  |
-| outband_network_segment | String | Yes |  |
-| server_templates | List | Yes | server template list included in the resource pool |
-| server_infos | List | Yes | server information list included in the resource pool |
+| RES_POOL_NAME | String | Yes | This is the unique name of the resource pool, could be refered by other parameters |
+| RES_POOL_TYPE | String | Yes | User defined value to identify different hardware or software configuration requirements. |
+
+<p align="center"><b>Table 6-3-1:</b> Resource Pool Information.</p>
 
-<p align="center"><b>Table 6-3-1:</b> Hardware resource description.</p>
 
 <a name="6.3.2"></a>
-### 6.3.2 Server template
-The server template describes the capability of the host like processor, memory, harddrive, raid, NIC/NIC binding, manufacturer, model etc.
-Server template would be assigned to multiple servers, i.e physical hosts.
+### 6.3.2 Global Settings
+The Global settings are provided by the user, contains data like like IP_Type, VLAN_Type, etc.
+
+Only one instance per resource pool.
 
 | Field # | type | mandatory | Instruction |
 |----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| template_name | String | Yes | template name will referenced by dedicated server |
-| manufacturer | String | Yes |  |
-| model | String | Yes |  |
-| processor | String | Yes |  |
-| memory | String | Yes |  |
-| hard_drive | String | Yes |  |
-| raid | String | Yes |  |
-| network_card_infos | List | No | interface list definition|
-| network_card_bond_infos | List | No | NIC bonding, might not be always the case.|
+| IP_TYPE | String | Yes | IPV4 or IPV6 |
+| NETWORK_TYPE | String | Yes | VLAN or VXLAN  |
+| TIMEZONE | String | Yes | The timezone where VIM deployed, like UTC+8  |
+| STORAGE_TYPE | String | Yes |  The Storage type, for example "ceph" |
+| HUGEPAGE_ENABLE | String | Yes | TRUE or FALSE  |
+| HUGEPAGE_SIZE | String | Yes | The storage size that hypervisor set for VM, for example "1GB" |
+| QINQ_ENABLE | String | Yes | TRUE or FALSE |
+| HYPERVISOR_CORES | String | Yes | number of CPU bounded for Hypervisor |
+| EXTERNAL_NTP_SERVER_IP | List | Yes | IP list of NTP server, seperated by comma, for example: primariy_IP;second_IP |
 
-<p align="center"><b>Table 6-3-2-1:</b> Server template.</p>
+<p align="center"><b>Table 6-3-2:</b> Global Settings </p>
 
-This is the network interface definition. Generally, there are a list of interfaces included in server template.
+
+<a name="6.3.3"></a>
+### 6.3.3 Parameters for network virtualization
+MTU value for network virtualization should be defined, this is usually standard value that defined by user.
+
+3 instances are expected for Manage Service and storage,  they may have different MTU requirement.
```

Comment:
"instances" -> instances of the abc?

Reply:
I have updated the description, could you help to check is it clear enough?